lib/jekyll/scholar/tags/bibliography.rb

```diff
@@ -29,7 +29,11 @@ def render(context)
           end
         end
 
-        items = items.take(max.to_i) if limit_entries?
+        # this also allows for --max {...,-2,-1,0,1,...}
+        if max > -1
+            @max = offset + max - 1
+        end
+        items = items[offset..max]
```

Comment:
I would prefer: `items = items[offset..max] if limit_entries?` -- if you adjust `limit_entries?`